README.md

```diff
@@ -77,7 +79,13 @@ Then you can run the tests:
 ```bash
 npm run test-examples
 ```
+warning:
+	WebGl EXT_frag_depth extension not supported in headless mode
 
+option in headless mode:
```

Comment:
Alternative wording:

Supported environment variables:
- `SCREENSHOT_FOLDER`: take a screenshot at the end of each test and save it in this folder. Example: `SCREENSHOT_FOLDER=/tmp/`
- `CHROME`: path to Chrome executable. If unspecified itowns will use the one downloaded during puppeteer install.
- `DEBUG`: run Chrome in a window with the debug tools open.
- `REMOTE_DEBUGGING`: run Chrome in headless mode and set up remote debugging. Example: `REMOTE_DEBUGGING=9222` will setup remote debugging on port 9222. Then start another Chrome instance, browse to `chrome://inspect/#devices` and add `localhost:9222` in `Discover network targets`.

Note: Chrome in headless mode doesn't support the WebGL EXT_frag_depth extension. So rendering may differ and some bugs can only be present in headless mode.

Reply:
ok thanks